README.md

```diff
@@ -9,194 +9,192 @@
+Hence, the standard currently consists of four parts:
 
-### Usage
+1. [API Part](https://fdc3.finos.org/docs/api/spec)
+2. [App Directory Part](https://fdc3.finos.org/docs/app-directory/overview)
+3. [Intents part](https://fdc3.finos.org/docs/intents/spec)
+4. [Context Data Part](https://fdc3.finos.org/docs/context/spec)
 
-FDC3 can be accessed either via the global window object:
+The specifications are informed by agreed [business use cases](https://fdc3.finos.org/docs/use-cases/overview),
+and implemented and used by leading [financial industry participants](https://fdc3.finos.org/community#type-all).
 
-```ts
-if (window.fdc3) {
-  const channel = await window.fdc3.getOrCreateChannel('myChannel')
-  channel.broadcast({ ... })
-}
-```
+See https://fdc3.finos.org for more information, including on [Compliance] and the [FDC3 charter], as well as a comprehensive [API Reference]. You can also take the free [FDC3 Training](https://www.edx.org/course/fdc3-interoperability-for-the-financial-desktop) for an introduction to FDC3's core concepts and usage.
 
-Or by importing relevant FDC3 functions with ES6 module syntax:
+[FDC3 Charter]: https://fdc3.finos.org/docs/fdc3-charter
+[Compliance]: https://fdc3.finos.org/docs/fdc3-compliance
+[API Reference]: https://fdc3.finos.org/docs/api/ref/DesktopAgent
 
-```ts
-import { getOrCreateChannel } from '@finos/fdc3'
+# Supported Platforms
 
-// will throw if window.fdc3 is not available
-const channel = await getOrCreateChannel('myChannel')
-channel.broadcast({ ... })
-```
+ - As an open standard, FDC3 can be implemented on any platform and in any language.
+ - All that is required is a "desktop agent" that supports the FDC3 standard, which is responsible for co-ordinating application interactions.
+ - Get started using FDC3 on the web with typescript by reading the [supported platforms](https://fdc3.finos.org/docs/supported-platforms) page.
 
-## Native
-The FDC3 standard does not define wire formats for communication. Hence, for native applications to be FDC3-enabled, they need to make use of a library (e.g. a DLL in .Net or Jar file in Java) that provides them with an implementation of the FDC3 API. FDC3-enabled native applications are therefore specific to particular desktop container frameworks (or other suitable environments) that provide the necessary libraries.
+## FDC3 npm module
 
-Despite this limitation, implementing support for FDC3 in a native application can allow it to interact with a wide variety of FDC3-enabled web applications.
+The FDC3 npm package does NOT provide a Desktop Agent implementation. Rather, it can by used by web applications to target operations from the API Specification in a consistent way. Each FDC3-compliant desktop agent that the application runs in, can then provide an implementation of the FDC3 API operations.
```

Comment:
@robmoffat Shall we add at the end:

"And Open Source Electron-based implementation of an FDC3 Desktop Agent is available [here](https://github.com/finos/electron-fdc3)".

Reply:
now added in the resources section, above 👍